### PR TITLE
[i18n] Update Occitan language

### PR DESCRIPTION
**Pull request by Quent-in**
New strings translated into Occitan

### CODE REVIEW

**Review by aledeg — APPROVED** [review]

**Comment by aledeg** [comment]
Thank you for your contribution

File: app/i18n/oc/admin.php [hunk 1]
```diff
@@ -122,7 +122,7 @@
 		'title' => 'Extensions',
 		'update' => 'Mesa a jorn disponibla',
 		'user' => 'Extensions utilizaire',
-		'version' => 'Version',	// TODO - Translation
+		'version' => 'Version',
```
Comment (Frenzie) [inline app/i18n/oc/admin.php:125]: So many difficult ones. ;-)

File: app/i18n/oc/user.php [hunk 2]
```diff
@@ -30,15 +30,15 @@
 		),
 	),
 	'password' => array(
-		'invalid' => 'The password is invalid.',	// TODO - Translation
+		'invalid' => 'Lo senhal es invalid.',
```
Comment (Frenzie) [inline app/i18n/oc/user.php:33]: Even invalid is straight Occitan? Wow. :+1:

**Review by Frenzie — APPROVED** [review]

**Comment by Alkarex** [comment]
Thanks @Quent-in 👍 
Could you please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md ?

File: app/i18n/oc/user.php (hunk 2 above)
Comment (Quent-in) [reply app/i18n/oc/user.php:33]: Yes and even stronger, look:
The password is empty => Lo senhal es void
Easy to decode when you're developer, right? ^^

**Pull request merged**